src/controllers/profileController.js

```diff
@@ -42,11 +42,32 @@ class Profile {
   }
 
   /**
-       * @description update user profile
-       * @param {object} req
-       * @param {object} res
-       * @returns {object} return object containing updated user profile
-       */
+   * @description fetch all the profiles
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} all available profiles
+   */
+  static async fetchProfiles(req, res) {
+    try {
+      const users = await User.findAll({
+        where: { role: 'user' },
+        attributes: ['userName', 'bio', 'image']
+      });
+      res.status(200).json({
+        message: `${users.length} Users found`,
+        data: users
+      });
+    } catch (err) {
+      // console.log(err);
```

Comment:
do not mute the error. I'd rather you throw the error, this is important for debugging purposes

Reply:
Okay...fixed!